src/musl-prepo.json

```diff
@@ -59,6 +112,21 @@
           "definitions":[
             {"digest":"461387b22c520e0c42df3c56ab2aa511","name":4,"linkage":"external","visibility":"hidden"} //"__set_thread_area"
           ]
+        },
+        "140eae3767a12b28780d48ef2e02a69e":{
+          "triple":0, //"x86_64-pc-linux-gnu-repo"
+          "definitions":[
+            {"digest":"d8866e5798e0a50c741e3f1cc110e343","name":5,"linkage":"external"}, //"__setjmp"
+            {"digest":"d8866e5798e0a50c741e3f1cc110e343","name":6,"linkage":"external"}, //"_setjmp"
+            {"digest":"d8866e5798e0a50c741e3f1cc110e343","name":7,"linkage":"external"} //"setjmp"
```

Comment:
Why are there three identical functions all called [_]*setjmp? (Same applies to [_]*longjmp below.) I realise that the original object file may have had one definition and two aliases, but (as noted elsewhere) the repo doesn't currently support aliases.

Reply:
Good point. I removed unused functions in the commit https://github.com/SNSystems/musl-prepo/pull/9/commits/a434172d9f773d451c638755703f6814bc86e275.